noxfile.py

```diff
@@ -72,3 +72,17 @@ def coverage(session: Session) -> None:
     """Generate the coverage data."""
     session.install("coverage[toml]", "codecov")
     session.run("coverage", "xml", "--fail-under=0")
+
+
+@session(python=python_versions)
+def docs(session: Session) -> None:
```

Comment:
same as above seems like this should use the other venv

Reply:
done